lua/litee/gh/ghcli/init.lua

```diff
@@ -24,34 +24,86 @@ local function debug_fmt_args(args)
     return "gh " .. cmd
 end
 
+local function copy_table(t)
```

Comment:
Just curious, nvim has some built in table related functions, did none of these accomplish what youre doing here?